### DISCUSSION TITLE
dashboard of counties on homepage

### DISCUSSION THREAD

**Discussion by MargeretNyaga** [lsc-hubs/kenya-hub#26] [General]
Provide a dashboard on the homepage so that the data is categorized per country, going down to the lowest level for easier accessibility 
Some tabs on the catalogue are not opening, like the county tab on the spatial scope tab did not categorize the data per preferred county name.
When downloading the final map on the Mapviewer, the final map does not contain the key or legend
There are misspelled words like soil organic carbon, which is written as "Oranic carbon"

**Comment by pvgenuchten** [comment]
thank you @MargeretNyaga for reporting these issues, 

- I like the idea of having a dashboard on the homepage, can you clarify (maybe with a example) what info you would like to see on the dashboard?

- Can you add a snapshot of the tabs which are not opening, this will help people to identify where you notice a improvement option (on windows, use [win]-[shift]-[s] combo to select an area and paste in this comment window)

- To export the current map, use the share/print option:
![image](https://github.com/lsc-hubs/kenya-hub/assets/299829/36d56b14-ffd5-48f5-8464-e0ef51896387)

- the `oranic` typo should be fixed now.
